io.github.idevecore.Valuta.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "io.github.idevecore.Valuta",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "valuta",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--own-name=io.github.idevecore.Valuta.SearchProvider"
```

Comment:
this shouldn't be needed.

Reply:
Hello, I removed the line "--own-name=io.github.idevecore.Valuta.SearchProvider".